nativelib/src/main/scala/scala/scalanative/posix/unistd.scala

```diff
@@ -16,8 +16,8 @@ object unistd {
 
   type off_t = CLongLong
 
-  def sleep(seconds: CUnsignedInt): Int                           = extern
-  def usleep(usecs: CUnsignedInt): Int                            = extern
+  def sleep(seconds: CUnsignedInt): CInt                          = extern
```

Comment:
Actually, this one is `unsigned int sleep(unsigned int);` in C, therefore it should return `CUnsignedInt`.